experimental/packages/opentelemetry-sdk-metrics-base/src/aggregator/types.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HrTime } from '@opentelemetry/api';
+import { Attributes } from '@opentelemetry/api-metrics';
+import { InstrumentationLibrary } from '@opentelemetry/core';
+import { Resource } from '@opentelemetry/resources';
+import { AggregationTemporality } from '../export/AggregationTemporality';
+import { Histogram, MetricData } from '../export/MetricData';
+import { InstrumentDescriptor } from '../InstrumentDescriptor';
+import { Maybe } from '../utils';
+
+/** The kind of aggregator. */
+export enum AggregatorKind {
+  NONE,
+  SUM,
+  LAST_VALUE,
+  HISTOGRAM,
+}
+
+/** Sum returns an aggregated sum. */
+export type Sum = number;
+
+/** LastValue returns last value. */
+export type LastValue = number;
+
+export type PointValueType = Sum | LastValue | Histogram;
```

Comment:
Just a nit: I couldn't find this being used anywhere. Are there plans for this? :thinking:

Reply:
Yeah, it's not used. Removed.